Lets Do This/Controllers/LDTRegisterViewController.h

```diff
@@ -12,6 +12,8 @@
 
 @property (nonatomic, strong) IBOutlet UITextField *emailField;
 @property (nonatomic, strong) IBOutlet UITextField *passwordField;
+@property (weak, nonatomic) IBOutlet UITextField *firstNameField;
+@property (weak, nonatomic) IBOutlet UITextField *lastNameField;
```

Comment:
You’re absolutely right. Before Xcode 4 you had to put them in the header file and apparently sometimes habits from those days creep in! Feel free to move them or I can as well since it was my mistake in the first place.

> On May 15, 2015, at 7:45 AM, Aaron Schachter <notifications@github.com> wrote:
> 
> In Lets Do This/Controllers/LDTRegisterViewController.h:
> 
> > @@ -12,6 +12,8 @@
> >  
> >  @property (nonatomic, strong) IBOutlet UITextField *emailField;
> >  @property (nonatomic, strong) IBOutlet UITextField *passwordField;
> > +@property (weak, nonatomic) IBOutlet UITextField *firstNameField;
> > +@property (weak, nonatomic) IBOutlet UITextField *lastNameField;
> 
> @ryangrimm I declared these here to keep things consistent, but would it be better practice to move all of these outlets into the .m file?
> 
> —
> Reply to this email directly or view it on GitHub.
>